### ISSUE TITLE
Function keys do not work as advertised in-game 

### ISSUE THREAD

**Issue by shirishag75** [mars-sim/mars-sim#46]
I ran the game on Lenovo Thinkpad T440 with 8 GB RAM. 

While I was able to see the map, the function keys as shared in-game don't work. F3 which should have simulated/clicked/shown the monitor tool didn't do  and was showing the brightness. Maybe I need to fix something in thinkpad but what I don't know.

**Comment by mokun** [comment]
> the function keys as shared in-game don't work. 

I use to have a thinkpad in the past. Be sure to use the top row key F1 to F7 on it so that they are not being "taken" by your bios as internal Function keys.

> F3 which should have simulated/clicked/shown the monitor tool didn't do and was showing the brightness.

F3 is Time Tool. It will soon be discontinued since the info on F3 will be shown in Mars Calendar Panel and the Speed Panel.

F4 is the Monitor Tool. One of these days, I'll recreate the data tables with JavaFX UI but it will take a while.

**Comment by larsnaesbye** [comment]
Yeah, when the Time Tool goes away, we might have to rethink all the shortcut keys. That is long overdue in fact :-)
